reconcile/sentry_config.py

```diff
@@ -0,0 +1,317 @@
+import logging
+import reconcile.queries as queries
+import utils.gql as gql
+import utils.secret_reader as secret_reader
+
+from reconcile.github_users import init_github
+from utils.config import get_config
+from utils.sentry_client import SentryClient
+
+
+SENTRY_PROJECTS_QUERY = """
+{
+  apps: apps_v1 {
+    sentryProjects {
+      team {
+        name
+      }
+      projects {
+        name
+        description
+        email_prefix
+        platform
+        sensitive_fields
+        safe_fields
+        auto_resolve_age
+        allowed_domains
+      }
+    }
+  }
+}
+"""
+
+SENTRY_USERS_QUERY = """
+{
+  roles: roles_v1 {
+    name
+    users {
+      name
+      github_username
+    }
+    bots {
+      name
+      github_username
+    }
+    permissions {
+      service
+      ...on PermissionSentryTeam_v1 {
+        team
+      }
+    }
+  }
+}
+"""
+
+SENTRY_TEAMS_QUERY = """
+{
+  teams: sentry_teams_v1 {
+    name
+  }
+}
+"""
+
+class SentryState:
+  def __init__(self):
+    # Map of user:teams[]
```

Comment:
I *really* like that you give the spec of the content of the data. I wish we would do this always! cc @maorfr @apahim